templates/open-enterprise/scripts/recover-dao.js

```diff
@@ -0,0 +1,95 @@
+/* global web3 */
+const fs = require('fs')
+const exec = require('child_process').exec
+const rl = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout
+})
+const OpenEnterpriseTemplate = fs.readFileSync('../build/contracts/OpenEnterpriseTemplate.json')
+
+const zeroAddress = '0x00000000000000000000000000000000000000000'
+
+let deployedInfo = null
+
+// This function queries APM for the contract address of the Open Enterprise template
+// The script will not exit until this address is resolve or the request fails or times out
+exec('aragon apm info open-enterprise-template --env production', (err, stdout, stderr) => {
+  if (err) {
+    console.error(err, ' error getting template address: ', stderr)
+    return
+  }
+  deployedInfo = JSON.parse(getJson(stdout))
+})
+
+module.exports = async function getSecondDaoTxData(callback) {
+  // This function will generate a data string that the user can send in a tx to our deployments contract
+  const oeAbi = JSON.parse(OpenEnterpriseTemplate).abi
+  const TemplateContract = web3.eth.contract(oeAbi).at(zeroAddress)
+  const voteDuration = await getVoteDuration()
+  const minSupport = await getMinSupport()
+  const quorum = await getQuorum(minSupport)
+  const allocationsPeriod = await getAllocationsPeriod()
+  const dataString = TemplateContract.newOpenEnterprise.getData(
+    [ minSupport, quorum, voteDuration ],
+    allocationsPeriod,
+    false
+  )
+  rl.write('generating information...')
+  while (!deployedInfo) {
+    await waitASecond()
+    rl.write('.')
+  }
+  rl.write(`\nHere's your data string:\n${dataString}\n\n`)
+  rl.write(`submit a transaction with the data string above to ${deployedInfo.contractAddress} in MyCrypto or MEW\n`)
+
+  callback()
+}
+
+const waitASecond = () => {
+  return new Promise((resolve) => {
+    setTimeout(resolve, 1000)
+  })
+}
+const getVoteDuration = () => {
+  return new Promise((resolve) => {
+    rl.question('enter the desired dot voting period length, in days > ', (answer) => {
```

Comment:
Why in days? we allow even seconds to be set, I would set it to seconds instead of limit the user, it takes literally more work to limit it to be days (because of the calculation), unless there is a good reason for this.

Reply:
I thought the average production user would be thinking in the timescale of days, but if you think seconds makes more sense we can do that.

Reply:
I don't think the average production user will ever use this without guided support from anyone of the Autark team, because of the complexity using this tool involves (node packages, cli commands, truffle, etc), in the way it is currently designed, I would think of us (Autark team) as the target user of it (while providing support to other people).

If you seriously think this must be a tool for production users, I would think in differently architecting some assumptions (as I briefly indicated in other comments) , like providing encapsulated binaries or even a GUI tool, but for now I really think, for the support scenario I mention, it is ideal in its state, and a perfect starting point to iterate from, in case we want it to evolve that way 👍 

That being said, it is obviously not strictly needed to change it from days to seconds, but I think even a production user could come up with a DAO idea where votes period is less than one day, I don't think it would be justified to limit something that is allowed in the contract, just because of the target user, because that would be providing unfair advantage to other people. My point is, limit everywhere or not limit at all... IMO all our tools should mimic 100% the contract logic, but please, again, consider this a very personal opinion! I understand your motivation for simplification!